### PR TITLE
Add support for PHP 8 drop support for PHP 7.1 and lower

### PR DESCRIPTION
**Pull request by Rafalsky**

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        php-versions: ["5.6", "7.0", "7.1", "7.2", "7.3"]
+        php-versions: ["7.1", "7.2", "7.3"]
```
Comment (jkowalleck) [inline .github/workflows/php.yml:12]: could you add a test for php7.4 and php8.0 ?

**Review by jkowalleck — CHANGES_REQUESTED** [review]

**Comment by jkowalleck** [comment]
hello @Rafalsky 

Thank you for the pull request, very much appreciated.

One slight remark: 
you removed php<7.1 from the github action. 
could you add php >7.3 to the github action in return?

**Comment by Rafalsky** [comment]
@jkowalleck Updated as requested.

**Review by jkowalleck — APPROVED** [review]

**Comment by jkowalleck** [comment]
@coderpatros 

i'd love to have this PR merged to master
we could release it as 1.3, since it does not include breaking changes in the interfaces CLI nor method calls.

what do you think?

----

It's common practice in the PHP SemVer world, to have minor versions that change the minimum requirements(`composer.json`)
Impact on the users: composer will automatically download compatible version suitable to the runtime environment, according to the users decision in thier `composer.json`.

see prominent examples introducing php8 support:
* phpunit:  via [9.3](https://packagist.org/packages/phpunit/phpunit#9.3.0)
  by changing from `php: ^7.3` to `php: ^7.3 || ^8.0`
* symfony: via [5.1.0](https://packagist.org/packages/symfony/framework-bundle#v5.0.9)
  by changing from `php: ^7.2.5` to `php: >=7.2.5`
* laravel: via [8.12.0](https://packagist.org/packages/laravel/framework#v8.12.0)
  by changing from `php: ^7.3` to `php: ^7.3 | ^8.0`

**Comment by coderpatros** [comment]
@jkowalleck I think if we are dropping support for a minor version of PHP we should probably treat that as a major version bump.

**Comment by jkowalleck** [comment]
@coderpatros  i see.

thanks again, @Rafalsky .
your changes will be in the upcoming 2.0 release, then.

**Pull request merged**